docs/docusaurus/docs/cloud/schedules/manage_schedules.md

```diff
@@ -0,0 +1,75 @@
+---
+sidebar_label: 'Manage schedules'
+title: 'Manage schedules'
+description: Create and manage schedules for Validations in GX Cloud.
+---
+
+Add schedules to your GX Cloud Expectation Suites to take advantage of automated data quality checks. Schedules are automatically enabled when you add your first Expectation to and Expectation Suite in the Cloud UI.
+
+## Create a schedule for an existing Expectation Suite
+
+1. In GX Cloud, click **Data Assets**.
+
+2. In the **Data Assets** list, click the Data Asset name.
+
+3. Click the **Expectations** tab and then select the Expectation Suite you'd like to apply a schedule to.
+
+4. Ensure you have an Expectation in the Expectation Suite. Unpause the schedule for the Expectation Suite using the toggle in the Scheduling component.
+
+5. To edit the default schedule, click the edit icon in the Scheduling component.
+
+    - **Frequency:** Select a frequency between 1 hour and 24 hours.
+
+    - **Start time:** Select a local start time for your schedule. Your selected frequency overrides start time, in the case where start time is later than the next occurence of selected frequency.
+
+6. Click **Save**.
+
+## Create a schedule for a new Expectation Suite
+
+1. In GX Cloud, click **Data Assets**.
+
+2. In the **Data Assets** list, click the Data Asset name.
+
+3. Click the **Expectations** tab.
+
+4. Click **New Suite** in the **Expectation Suites** pane.
+
+5. Enter a name for the Expectation Suite in the **Expectation Suite name** field.
+
+6. Click **Create**.
+
+7. Add an Expectation to the Expectation Suite to unpause the schedule for the Expectation Suite.
```

Comment:
This still seems like a strange behavior to me.  What if I'm in the process of building an Expectation Suite in GX Cloud and don't want to bloat my Validation Results until I'm done, but that process is going to take a few days for some reason?  Wouldn't I end up with Validation Results for the incomplete Expectation Suite?  Is there a way to disable the scheduler until I'm ready for it to start without backing out of creating the Expectation Suite and following the "Disable a schedule" procedure?

Reply:
There's no other path at the moment– we don't expect people who create expectations in the UI to be working on them for an extended period of time. In the next few months, we will also be moving to a simpler user experience where schedules are attached individually to expectations, which should remedy this.